packages/sitecore-jss/package.json

```diff
@@ -46,6 +47,7 @@
   },
   "dependencies": {
     "lodash.unescape": "^4.0.1",
+    "memory-cache": "^0.2.0",
```

Comment:
Is there a reason [memory-cache](https://www.npmjs.com/package/memory-cache) was used over [node-cache](https://www.npmjs.com/package/node-cache)? The node-headless-ssr-proxy sample uses `node-cache` for dictionary caching, `memory-cache` for general request middleware (not sure why the 2 caching packages are necessary). Seems like `node-cache` is more popular/active?

Reply:
@ambrauer Agree, changed to `node-cache`